src/nl/hannahsten/texifyidea/run/sumatra/SumatraConversation.kt

```diff
@@ -89,14 +118,15 @@ class SumatraConversation : ViewerConversation() {
      * Open a file in SumatraPDF, starting it if it is not running yet.
      */
     fun openFile(pdfFilePath: String, newWindow: Boolean = false, focus: Boolean = false, forceRefresh: Boolean = false, sumatraPath: String? = null) {
+        openConversation()
```

Comment:
Initialization should be done in the execute() method such that a connection is always opened if not yet open (e.g. what happens if you execute forwardSeach() before openFile() )

Reply:
Indeed, done.